src/less/details/ds6/details.less

```diff
@@ -0,0 +1,36 @@
+@import "../../less/ds6/variables.less";
+
+details:not([open]) {
```

Comment:
We'll need to use a class, rather than global style. So this should be `details.details` and `summary.details__summary`, etc.

Reply:
I assume the `expand-btn__icon` should also be changed to `expand-details__icon`?